app/views/announcements/_new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='card bg-light col-10 rounded mt-4 shadow-sm mx-auto'>
+    <div class='card-body'>
+        <h4>Make an announcement</h4>
+        <%= form_for @announcement do |f| %>
+        
+          <%= f.hidden_field :id, :value => @announcement.id %>
+          
+          <%= f.hidden_field :user_id, :value => current_user.id %>
+          
+          <%= f.text_area :text, placeholder: 'Announcement body', class: 'form-control mb-2 shadow' %>
+          
+          <%= f.select :placement_id, 
+            @placements.collect { |p| [p.name, p.id]}, 
+            :prompt => 'Placement (optional)' %> 
```

Comment:
The problem with this is if someone clicks on it, and they did not mean to specify a placement, they won't be able to get it back to blank.  It seems as if you would want an :include_blank => true in the f.select statement.  (I think what I'm saying is correct -- please respond if not.)